.github/workflows/workflow.yaml

```diff
@@ -7,6 +7,11 @@ jobs:
     runs-on: macos-latest
     steps:
       - uses: actions/checkout@v2
+      - name: Mount Bazel cache
+        uses: actions/cache@v1
+        with:
+          path: ~/repo-cache
+          key: repo-cache-${{ secrets.REPO_CACHE_VERSION }}
```

Comment:
Why a secret instead of a suffix configured in the `workflow.yaml`? IIUC this will need to be updated manually whenever the external dependencies are changed, so might just as well update it in the corresponding PR.

Also, why rely on a manual version bump, that is likely to be forgotten, instead of using something like [`hashFiles`](https://docs.github.com/en/free-pro-team@latest/actions/guides/caching-dependencies-to-speed-up-workflows#example-using-multiple-restore-keys) or commit sha with fall-through?

Reply:
Because this is what is recommended by GitHub staff: https://github.community/t/how-to-clear-cache-in-github-actions/129038/5.

We could spend brain cycles working out exactly which files should go into `hashFiles()`, but even if we get that correct now we would have to make sure that list stays correct later. Hence not obviating the need for a manual mechanism to purge the cache, as above. Setting a cache version number is expedient and I submit sufficient given the short-lived nature of caches on GitHub anyways (7 days).

Reply:
Fair enough, though the way I read that thread it's not a direct recommendation of this approach and more a mention that it exists. I'm all on-board for minimizing brain cycles on this aspect. But, I think that speaks in favor of a simple suffix in the code instead of requiring an out-of-band change through GH secrets. Case in point, I myself don't see the corresponding secret in the repository's secrets tab, so there's already some permissions issue to debug, and external contributors won't be able to access it for sure. I've opened a PR here: https://github.com/tweag/rules_haskell/pull/1472

Reply:
FWIW you do have access to that secret. It just doesn't exist right now. I added it then removed it as an experiment.